internal/commands/error.go

```diff
@@ -13,5 +13,41 @@ func HandleError(exec Executor, key string, err error) (output.Output, error) {
 		Details: "Error: " + err.Error(),
 	})
 
-	return nil, err
+	return nil, handledError{err}
+}
+
+type iHandledError interface {
+	HasBeenLogged()
+}
```

Comment:
Do we need this interface if we just check `err.(handledError)` in `outputError` function ?

Reply:
Good point! Removed this extra interface.